internal/characters/zhongli/stele.go

```diff
@@ -63,7 +63,7 @@ func (c *char) newStele(dur int, max int) {
 	c.steleSnapshot = combat.AttackEvent{
 		Info:        aiSnap,
 		Snapshot:    snap,
-		Pattern:     combat.NewCircleHit(c.Core.Combat.Player(), 5.66),
+		Pattern:     combat.NewBoxHitOnTarget(c.Core.Combat.Player(), combat.Point{Y: -4}, 8, 8),
```

Comment:
This offset doesn't make sense for how resonance ticks in game

When stele created, it should be given the player direction. This hitbox should be centered on stele + rotated towards create/player direction?

Reply:
I agree that this doesn't make sense and doesn't mirror the game, but that's how it's in the files. I'll change this to use the actual stele pos where the offset is `combat.Point{Y: 3}`.

Reply:
After further discussion, this will use Y: -1 considering box centering.